inventory/management/misc/batch_transfers.rst

```diff
@@ -0,0 +1,126 @@
+=======================
+Process Batch Transfers
+=======================
+
+Batch picking allows a single picker to handle a batch of orders,
+reducing the number of times he must visit the same location. In Odoo,
+it means you will regroup several transfers into the same batch
+transfer, then processing it via its form view or the barcode
+application.
+
+Create a Batch Transfer
+=======================
+
+To activate the batch picking option, go to :menuselection:`Inventory
+--> Configuration --> Configuration` and enable *Batch Pickings*.
+
+.. image:: media/batch_transfers_01.png
+    :align: center
+    :alt: View of the inventory settings. Process to enable the batch pickings in the Odoo Inventory app
+
+Then, go to :menuselection:`Inventory --> Operations --> Batch
+Transfers` and hit the create button.
+
+.. image:: media/batch_transfers_02.png
+    :align: center
+    :alt: View of the new menu, Batch Transfers, under operations
+
+Now, determine the batch transfer responsible and the type of transfers
+you want to include in the batch. To do so, click on *Add a line*.
+
+.. image:: media/batch_transfers_03.png
+    :align: center
+    :alt: View of a Batch Transfers form
+
+For example, use the filter feature to only have the *Pick* step
+before shipping the products to customers. After that, select the
+different transfers which should be included in the batch.
```

Comment:
```suggestion
In the example below, a filter was applied in order to only see the transfers* that are in the *Pick* step, before shipping the products to customers. After that, the
different transfers that** should be included in the batch were selected.
```

* (don't know if this is the right word) ^^
** I'd use 'that' because I believe it is restrictive clause (:

Reply:
I changed the sentence.